pkg/autodiscover/autodiscover.go

```diff
@@ -190,22 +197,23 @@ func namespacesListToStringList(namespaceList []configuration.Namespace) (string
 func getOpenshiftVersion(oClient clientconfigv1.ConfigV1Interface) (ver string, err error) {
 	var clusterOperator *configv1.ClusterOperator
 	clusterOperator, err = oClient.ClusterOperators().Get(context.TODO(), "openshift-apiserver", metav1.GetOptions{})
-	// error here indicates logged in as non-admin, log and move on
 	if err != nil {
 		switch {
-		case kerrors.IsForbidden(err), kerrors.IsNotFound(err):
```

Comment:
Why not keep reporting this error but this time as an error instead of just log?

Reply:
Not sure what you mean exactly here, can you elaborate, please?

Reply:
I was just suggesting to return an error for IsForbidden cases

Reply:
never mind, it is already returned in default : !